### ISSUE TITLE
Cannot serialize worker response

### ISSUE THREAD

**Issue by RobbieTheWagner** [ember-template-lint/ember-template-lint-plugin-prettier#289]
I am getting an error running `lint:hbs` that says `Cannot serialize worker response`. If I remove this plugin, it goes away. Any ideas?

Reproduction here https://github.com/shipshapecode/swach/pull/1947

**Comment by RobbieTheWagner** [comment]
@bertdeblock @dcyriller @patricklx I am still hitting this issue. Anyone have any ideas?

**Comment by bertdeblock** [comment]
Nope, sorry, we don't use `ember-template-lint-plugin-prettier` anymore.
